client/src/commons/CheckBox/CustomCheckbox.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { AriaAttributes, DOMAttributes } from 'react';
```

Comment:
where do you use this?

Reply:
I used this previously, then reverted the changes and forgot to remove the import.

Reply:
so remove

Reply:
Done.